src/integration-tests/src/storage.rs

```diff
@@ -102,3 +146,14 @@ async fn cleanup_bucket(client: storage::client::Storage, name: String) -> Resul
     let _ = futures::future::join_all(pending).await;
     client.delete_bucket(&name).send().await
 }
+
+pub(crate) fn random_bucket_id() -> String {
+    const CHARSET: &[u8] = b"abcdefghijklmnopqrstuvwxyz0123456789";
+    const PREFIX: &str = "rust-sdk-testing-";
```

Comment:
If we use the `integration-tests` label we do not need a long prefix like this.

Reply:
I like the prefix though. It makes it more obvious what these things are if they leak (especially in a testing project). I don't mind sacrificing some bits of randomness.